yarn.lock

```diff
@@ -0,0 +1,6179 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Is it really necessary to commit the auto-generated file? Guess it is better to add it to `.gitignore` otherwise will be always local changes and conflicts.